packages/ngtools/webpack/src/transformers/replace_resources.ts

```diff
@@ -98,7 +98,7 @@ function visitComponentMetadata(
       return ts.updatePropertyAssignment(
         node,
         ts.createIdentifier('template'),
-        createRequireExpression(node.initializer),
+        createRequireExpression(node.initializer, '!raw-loader!'),
```

Comment:
Shouldn't this be done when `directTemplateLoading` ?

Reply:
I was assuming this would still be needed for the case 'directTemplateLoading=false'. Or do we only want to support this with directTemplateLoading enabled? This would make the PR much simpler, I would then just remove the changes I made to replace_resources.

Reply:
I think it should be when it's `true`, let's confirm with @clydin.

Reply:
The check should actually be in both places (and should have been there for the initial direct template loading implementation).
So yes; please add a guard here based on the value of `directTemplateLoading`.
This is to ensure that custom Webpack configurations that use the plugin directly can use their own custom loader configurations to handle templates, if desired.